### DISCUSSION TITLE
Is there any way to add all the rules at the same time?

### DISCUSSION THREAD

**Discussion by CarlosParrasMartin** [rectorphp/rector#8443] [Q&A] [answered]
I want to use all the rules, but i don't want to go 1 by 1 adding them manually, is there any way to add them all at once or should i have to add them manually?

**Comment by samsonasik** [comment, marked as answer]
rules is part of set list, you can use `sets()` to enable the collection of `rules`, see documentation https://getrector.com/documentation/set-lists

**Reply by CarlosParrasMartin** [reply]
And how could i exclude in this way a rule if i want a especific rule not to be executed?
